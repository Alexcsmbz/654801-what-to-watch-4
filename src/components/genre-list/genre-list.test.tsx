import GenreList from './genre-list';
import GenreListItem from 'components/genre-list-item/genre-list-item';

const genres = [
  `Comedy`,
  `Drama`,
  `Crime`,
];

describe(`GenreList snapshot test`, () => {
  it(`GenreList should render genre list`, () => {
    const tree = renderer.create(
        <GenreList>
          {genres.map((g) => <GenreListItem key={g} item={g} />)}
        </GenreList>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
