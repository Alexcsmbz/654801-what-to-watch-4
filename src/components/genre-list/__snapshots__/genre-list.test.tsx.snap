// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenreList snapshot test GenreList should render genre list 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item undefined"
  >
    <div
      className="catalog__genres-link"
    >
      Comedy
    </div>
  </li>
  <li
    className="catalog__genres-item undefined"
  >
    <div
      className="catalog__genres-link"
    >
      Drama
    </div>
  </li>
  <li
    className="catalog__genres-item undefined"
  >
    <div
      className="catalog__genres-link"
    >
      Crime
    </div>
  </li>
</ul>
`;
